src/web/routes/application/steps/address/postcode/os-places.js

```diff
@@ -7,13 +7,13 @@ const OS_PLACES_API_PATH = '/places/v1/addresses/postcode'
 
 const standardisePostcode = (postcode) => postcode.toUpperCase().replace(/\s/g, '')
 
-const getAddressLookupResults = async (config, postcode) => {
+const getAddressLookupResults = async (config, postcode, language) => {
```

Comment:
You don't need to pass `language` this into this method.

Reply:
That's removed now.